docs/overview/queries/elastic_query_terms_set.md

```diff
@@ -0,0 +1,40 @@
+---
+id: elastic_query_terms_set
+title: "Terms Set Query"
+---
+
+The `TermsSet` query returns documents that contain the minimum amount of exact terms in a provided field. The terms set query is the same as [[zio.elasticsearch.query.TermsQuery]], except you can define the number of matching terms required to return a document.
+
+In order to use the `TermsSet` query import the following:
+```scala
+import zio.elasticsearch.query.TermsSetQuery
+import zio.elasticsearch.ElasticQuery.termsSetQuery
+```
+
+You can create a `TermsSet` query with defined `minimumShouldMatchField` using the `termsSet` method this way:
+```scala
+val query: TermsSetQuery = termsSet(field = "stringField", minimumShouldMatchField = "intField", terms = "a", "b", "c")
+```
+
+You can create a [type-safe](https://lambdaworks.github.io/zio-elasticsearch/overview/overview_zio_prelude_schema) `TermsSet` query with defined `minimumShouldMatchField` using the `termsSet` method this way:
+```scala
+val query: TermsSetQuery = termsSet(field = Document.name, minimumShouldMatchField = "intField", terms = 1, 2, 3)
+```
+
+You can create a `TermsSet` query with defined `minimumShouldMatchScript` using the `termsSetScript` method this way:
+```scala
+val query: TermsSetQuery = termsSetScript(field = "stringField", minimumShouldMatchScript = Script("doc['intField'].value"), terms = 1, 2, 3)
```

Comment:
Put `Script` import here.

Reply:
Also because of stringField, consider changing 1,2,3 to something else (everywhere here)